recipes/libmeshb/all/test_package/test_package.c

```diff
@@ -0,0 +1,8 @@
+#include <libmeshb7.h>
+
+#include <stdio.h>
+
+int main() {
+   GmfCloseMesh(NULL);
+   return 0;
+}
```

Comment:
```suggestion
#include <libmeshb7.h>

int main() {
   int ver, dim;
   int64_t LibIdx = GmfOpenMesh("/dev/null", GmfRead, &ver, &dim);
   return 0;
}

```
Let's try something that does not raise a signal. I tested locally, in case not be able to read the file (permission or does not exist), it will return -1. As I commented before, we just want to validate the package (header include and library linkage), the test package does not need be functional.

Reply:
Reading `/dev/null` probably won't work on Windows, though. Might as well try `""` probably.

Reply:
No problem at all. Like I commented above, missing file will result in failure as well. This current example fails in Linux already, we just do not check anything. :smiling_face_with_tear:

Reply:
Also tested using a non-existing file in Linux, like `/foo/bar` and resulted in the same.